test/powershell/SDK/PSDebugging.Tests.ps1

```diff
@@ -184,5 +185,94 @@ Describe "PowerShell Command Debugging" -tags "CI" {
 
 }
 
+# Scripting\Debugging\RunspaceDebuggingTests.cs
+Describe "Runspace Debugging API tests" -tag CI {
+    Context "PSStandaloneMonitorRunspaceInfo tests" {
+        BeforeAll {
+            $runspace = [runspacefactory]::CreateRunspace()
+            $runspaceType = [PSMonitorRunspaceType]::WorkflowInlineScript
+            $monitorInfo = [PSStandaloneMonitorRunspaceInfo]::new($runspace)
+            $instanceId = $runspace.InstanceId
+            $parentDebuggerId = [guid]::newguid()
+            $ps = [powershell]::Create()
+            $embeddedRunspaceInfo = [PSEmbeddedMonitorRunspaceInfo]::New($runspace,$runspaceType,$ps, $parentDebuggerId)
+        }
+        AfterAll {
+            $runspace.Dispose()
+            $ps.dispose()
+        }
+        
+        It "PSStandaloneMonitorRunspaceInfo should throw when called with a null constructor" {
```

Comment:
sure - I can make the language clearer

---
In reply to: [78447508](https://github.com/PowerShell/PowerShell/pull/2239#discussion_r78447508) [](ancestors = 78447508)